locale/circuitpython.pot

```diff
@@ -1956,18 +1956,34 @@ msgstr ""
 msgid "The length of rgb_pins must be 6, 12, 18, 24, or 30"
 msgstr ""
 
+#: shared-module/audioeffects/Echo.c
+msgid "The sample's bits_per_sample does not match the effect's"
+msgstr ""
+
 #: shared-module/audiomixer/MixerVoice.c
 msgid "The sample's bits_per_sample does not match the mixer's"
 msgstr ""
 
+#: shared-module/audioeffects/Echo.c
+msgid "The sample's channel count does not match the effect's"
+msgstr ""
+
 #: shared-module/audiomixer/MixerVoice.c
 msgid "The sample's channel count does not match the mixer's"
 msgstr ""
 
+#: shared-module/audioeffects/Echo.c
+msgid "The sample's sample rate does not match the effect's"
+msgstr ""
+
```

Comment:
It's probably better to drop "the effect's" and "the mixer's" so we can share these messages.